std/math.d

```diff
@@ -5589,7 +5589,7 @@ real NaN(ulong payload) @trusted pure nothrow @nogc
     }
 }
 
-pure nothrow @nogc unittest
+@safe pure nothrow @nogc unittest
```

Comment:
This broke the build for ARM.
```
../../../../src/libphobos/src/std/math.d:5643:32: error: cannot take address of local x in @safe function __unittestL5638_68
         auto xl = *cast(ulong*)&x;
                                ^
```
See https://github.com/D-Programming-GDC/GDC/blob/master/libphobos/src/std/math.d#L5638-L5647